components/collective-page/CollectiveNotificationBar.js

```diff
@@ -86,9 +86,6 @@ const getNotification = (intl, status, collective, host) => {
  * Adds a notification bar for the collective.
  */
 const CollectiveNotificationBar = ({ intl, status, collective, host }) => {
-  // Quickfix for events (as they are not inheriting isApproved status)
-  if (collective.type === CollectiveType.EVENT) return null;
-
```

Comment:
Not needed anymore since https://github.com/opencollective/opencollective-api/pull/3169 adds `isApproved` to events